tests/hardhat/Fork/supply.ts

```diff
@@ -0,0 +1,325 @@
+
+      // Mine 300,000 blocks
+      await mine(blocksToMint);
+
+      // Accural all the interest till latest block
+      await vUSDT.accrueInterest();
+
+      // Assert current exchange rate
+      exchangeRate = await vUSDT.callStatic.exchangeRateCurrent();
+      calculatedRate = await calculateExchangeRate();
+      expect(exchangeRate).closeTo(calculatedRate, 1);
+
+      // Setup for healAccount(acc2)
+      await priceOracle.setDirectPrice(usdd.address, convertToUnit(1, 10));
+      await priceOracle.setDirectPrice(usdt.address, convertToUnit(1, 18));
+
+      const [err2, liquidity2, shortfall2] = await comptroller.callStatic.getAccountLiquidity(acc2);
+      expect(err2).equals(0);
+      expect(liquidity2).equals(0);
+      expect(shortfall2).greaterThan(0);
+      await comptroller.setMinLiquidatableCollateral(convertToUnit(1, 22));
+
+      result = comptroller.connect(acc1Signer).healAccount(acc2);
+      await expect(result).to.emit(vUSDT, "RepayBorrow");
+
+      // Accural all the interest till latest block
+      await vUSDT.accrueInterest();
+
+      // Assert current exchange rate
+      exchangeRate = await vUSDT.callStatic.exchangeRateCurrent();
+      calculatedRate = await calculateExchangeRate();
+      expect(exchangeRate).closeTo(calculatedRate, 1);
+
+      const totalBal = await vUSDT.balanceOf(acc1);
+      await expect(vUSDT.connect(acc1Signer).redeem(totalBal)).to.emit(vUSDT, "Redeem");
+
+      // Mine 300,000 blocks
+      await mine(blocksToMint);
+
+      // Accural all the interest till latest block
+      await vUSDT.accrueInterest();
+
+      // Assert current exchange rate
+      exchangeRate = await vUSDT.callStatic.exchangeRateCurrent();
+      calculatedRate = await calculateExchangeRate();
+      expect(exchangeRate).closeTo(calculatedRate, 1);
+    });
+
+    it("Two users Mint, one redeems", async () => {
+      const mintAmount = convertToUnit("1", 18);
+      // Mint vUSDT with first account(acc1)
+      await usdt.connect(acc1Signer).allocateTo(acc1, convertToUnit(2, 18));
+      await usdt.connect(acc1Signer).approve(vUSDT.address, convertToUnit(2, 18));
+      await expect(vUSDT.connect(acc1Signer).mint(convertToUnit(1, 18))).to.emit(vUSDT, "Mint");
+
+      // Mint vUSDT with second account(acc2)
+      await usdt.connect(acc2Signer).allocateTo(acc2, convertToUnit(2, 18));
+      await usdt.connect(acc2Signer).approve(vUSDT.address, convertToUnit(2, 18));
+      await expect(vUSDT.connect(acc2Signer).mint(convertToUnit(1, 18))).to.emit(vUSDT, "Mint");
+
+      // Mint vUSDD with second account(acc2)
+      await usdd.connect(acc3Signer).faucet(mintAmount);
+      await usdd.connect(acc3Signer).approve(vUSDD.address, mintAmount);
+      await expect(vUSDD.connect(acc3Signer).mint(mintAmount)).to.emit(vUSDD, "Mint");
+
+      // Borrow usdt with second account(acc2)
+      await expect(vUSDT.connect(acc3Signer).borrow(usdtBorrowAmount)).to.be.emit(vUSDT, "Borrow");
```

Comment:
In the comments we say "second account", but we are really using the acc3

Reply:
Fixed, 4651fb2039bb0f5ad7bc667b2269087b10f7b6f0